src/main.ts

```diff
@@ -2,19 +2,19 @@ import * as path from "path";
 
 import { app } from "electron";
 
-import { BiometricMain } from "jslib-common/abstractions/biometric.main";
 import { StateFactory } from "jslib-common/factories/stateFactory";
 import { GlobalState } from "jslib-common/models/domain/globalState";
 import { StateService } from "jslib-common/services/state.service";
-import { KeytarStorageListener } from "jslib-electron/keytarStorageListener";
 import { ElectronLogService } from "jslib-electron/services/electronLog.service";
 import { ElectronMainMessagingService } from "jslib-electron/services/electronMainMessaging.service";
 import { ElectronStorageService } from "jslib-electron/services/electronStorage.service";
 import { TrayMain } from "jslib-electron/tray.main";
 import { UpdaterMain } from "jslib-electron/updater.main";
 import { WindowMain } from "jslib-electron/window.main";
 
-import { MenuMain } from "./main/menu.main";
+import { BiometricMain } from "./main/biometric/biometric.main";
```

Comment:
let's revert all these biometric changes until we actually implement them in our native module

Reply:
I needed to move biometric and keytar logic from jslib to avoid having to update directory connector.